containers/podman-desktop
# fix: zoom level resets on window change after navigation

### What does this PR do?

Switching routing mode to memory fixes the issue and according the doc here https://github.com/AlexxNB/tinro?tab=readme-ov-file#navigation-method

`By default, navigation uses the History API which allows you to have clean page URLs, although it needs some setup on the server side. Instead, you may choose to use hash or memory navigation methods.

There is no need to change links or paths in your app, everything else will still work the same.`

### Screenshot / video of UI

Before


https://github.com/user-attachments/assets/d8330e76-1c06-475f-80ed-cd0ba8b6a0ff

after


https://github.com/user-attachments/assets/3d202ce3-cab6-41b6-b4f2-84c0cfc3d2d2


### What issues does this PR fix or reference?

Fix #5265.

### How to test this PR?

Run in terminal to pull all relevant branches:

```
git remote add dg https://github.com/dgolovin
git remote fetch dg
git checkout i5265-switch-to-memory-mode
git pull dg i5265_navigation_fix --rebase
git pull dg allow-to-run-in-electron --rebase
```

Install dependencies, build and run podman-desktop in electron

```
yarn
yarn build
node_modules/.bin/electron .
```

When podman desktop window shows up zoom up window content and alternate page navigation and window resize. There should be no zoom changes.
